ci/ci.yml

```diff
@@ -150,25 +151,25 @@ Resources:
             Statement:
               - Effect: Allow
                 Action:
-                  - "ecr:GetAuthorizationToken"
-                  - "ecr:CreateRepository"
+                  - ecr:GetAuthorizationToken
+                  - ecr:CreateRepository
                 Resource: "*"
               - Effect: Allow
                 Action:
-                  - "ecr:*RepositoryPolicy"
-                  - "ecr:BatchCheckLayerAvailability"
-                  - "ecr:GetAuthorizationToken"
-                  - "ecr:InitiateLayerUpload"
-                  - "ecr:CompleteLayerUpload"
-                  - "ecr:UploadLayerPart"
-                  - "ecr:PutImage"
+                  - ecr:*RepositoryPolicy
```

Comment:
Hmm... yeah, i guess unquoted `*` is fine as long as the string doesn't start with it.